pkg/apis/projectcalico/v3/policy_common.go

```diff
@@ -144,6 +144,16 @@ type EntityRule struct {
 	// endpoints across all namespaces.
 	NamespaceSelector string `json:"namespaceSelector,omitempty" validate:"omitempty,selector"`
 
+	// ServiceSelector is an optional field that contains options for matching Kubernetes Services.
+	// If specified, only traffic that originates from or terminates at endpoints within the selected
+	// service(s) will be matched, and only to/from each endpoint's port.
+	//
+	// ServiceSelector cannot be specified on the same rule as Selector, NotSelector, NamespaceSelector, Ports,
+	// NotPorts, Nets, NotNets or ServiceAccounts.
+	//
+	// Only valid on egress rules.
+	Services *ServiceMatch `json:"services,omitempty" validate:"omitempty"`
```

Comment:
The comments seem to indicate that this should be named `ServiceSelector` as opposed to `Services`.

Reply:
oh right